src/_TextInputWrapper.tsx

```diff
@@ -27,16 +27,22 @@ const sizeVariants = variant({
   prop: 'size',
   variants: {
     small: {
+      '--inner-action-size': '20px',
       minHeight: '28px',
       px: 2,
       py: '3px',
       fontSize: 0,
       lineHeight: '20px',
     },
+    medium: {
+      '--inner-action-size': '24px',
+    },
     large: {
+      '--inner-action-size': '28px',
```

Comment:
I assume we picked `'28px'` because it's `primer.control.small.size` in [size primitives](https://primer.style/primitives/spacing#small). Is that right?